### ISSUE TITLE
Accessibility: datepicker with screen reader

### ISSUE THREAD

**Issue by hamsterbacke81** [primefaces/primereact#7978]
Right now. It is not really intuitive or understandable what screen reader read out about datepicker. In the following example, screen reader (in my case NVDA) would read "M column 3 14" M: abbreviation for Mittwoch(Wednesday in german), column since the calendar is built within a table. It is not really usable like this, is there a way to improve this?

![Image](https://github.com/user-attachments/assets/ed8e7585-1acb-4fbd-bc25-472a13e801c4)

**Comment by melloware** [comment]
@hamsterbacke81 when i browse the HTML for the Calendar it has `aria-label="05/15/2025"` which is what the screenreader should be reading?

![Image](https://github.com/user-attachments/assets/e320b724-7837-4d88-b2a3-ed4c598d6698)

**Comment by hamsterbacke81** [comment]
@melloware hm... just tested it with NVDA and the showcases here https://www.primefaces.org/showcase/ui/input/datepicker/datePicker.xhtml?jfwid=9e93a

the aria-label says "May 8", but the reader still reads "T h column 5 8". I don't have any other screen reader to test with...

**Comment by melloware** [comment]
You accidentally opened this in PrimeReact.   Moving to PrimeFaces...
